addons/skin.estouchy/xml/Includes.xml

```diff
@@ -1037,7 +1037,7 @@
 		</item>
 		<item id="3">
 			<label>20342</label>
-			<onclick>ActivateWindow(Videos,MovieTitles,Return)</onclick>
+			<onclick>ActivateWindow(Video,MovieTitles,Return)</onclick>
```

Comment:
Points to the wrapper
https://github.com/xbmc/xbmc/blob/master/xbmc/input/ButtonTranslator.cpp#L301